### ISSUE TITLE
Placeholder pic replacements

### ISSUE THREAD

**Issue by njsfield** [InFact-coop/our-site#2]
[These](https://github.com/InFact-coop/our-site/tree/master/images) image assets are placeholder background content currently.

Perhaps it would be worth grabbing alternatives from a site like [Shutter Stock](https://www.shutterstock.com/)?

**Comment by njsfield** [comment]
Shutter Stock has great images, though prices start from fixed **£29** for five images, and monthly subscription options from **£19**. Will explore some no-cost royalty free options online....

**Comment by emilyb7** [comment]
what do we want images of?

**Comment by njsfield** [comment]
I think currently the font is well defined thanks to the dark blurred placeholders, so anything livelier could be distracting. 

As an alternative, I created the branch [colour-experiment](https://github.com/InFact-coop/our-site/tree/color-experiment) to set *static colours* for the backgrounds, and set full viewport heights for each slide (so that each slide is always 100% height on mobile/tablet/desktop).

**Comment by emilyb7** [comment]
@njsfield i actually really like the current placeholders! But maybe we should ask some people who don't know that they're placeholders what they think?

If we can some quite subtle images to go in the background, that would be nice. But I would rather use the existing ones than spend money on stock images that have no meaning to us.

**Comment by njsfield** [comment]
Yeah I agree, they work well with the heading text and the colours suit the rest of the site nicely-
So finding replacements that are aesthetically similar but more contextually related to InFact could be tricky...

Agreed to wait to see what others think 👍
